components/treeview/stories/treeview.stories.js

```diff
@@ -15,14 +15,22 @@ export default {
 	component: "TreeView",
 	argTypes: {
 		items: { table: { disable: true } },
-		variant: { table: { disable: true } },
+		variant: {
+			name: "Variant",
+			type: { name: "string" },
+			category: "Component",
+			defaultValue: { summary: "default" },
+			options: ["default", "detached", "thumbnail"],
+			control: "inline-radio",
```

Comment:
This won't prevent me from approving, I just wanted to bring it up. Most of our instances of `inline-radio`s are 2 options (although one control in typography and tooltip don't do that). Should we do a select instead? 

I don't have particularly strong feelings either way, I feel like I read that comment somewhere to change from an inline-radio to a select once we added more options.

Reply:
Select feels more logical/natural to me, just pushed that change!